openmdao/core/system.py

```diff
@@ -2838,7 +2838,7 @@ def _list_states(self):
 
         return states
 
-    def add_recorder(self, recorder):
+    def add_recorder(self, recorder, recurse=False):
         """
         Add a recorder to the driver.
```

Comment:
Need to add the new arg to the docstring.

Reply:
Sorry. Fixed. BTW, shouldn't a test have failed ? I thought one of the tests checked for missing items in the docstring. I did run testflo before pushing this up

Reply:
And maybe not in this PR, but include explanation of this in your forthcoming recording feature doc branch.